lib/manageiq/automation_engine/logger.rb

```diff
@@ -1,6 +1,20 @@
 module ManageIQ
   module AutomationEngine
     class Logger < ManageIQ::Loggers::Base
+      attr_reader :wrapped_logger
+
+      def initialize(*args, **kwargs)
+        @wrapped_logger = kwargs.fetch(:wrapped_logger)
+        super(*args, **kwargs.except(:wrapped_logger))
```

Comment:
Minor, but I think you can do:

```suggestion
        @wrapped_logger = kwargs.delete(:wrapped_logger)
        super(*args, **kwargs)
```

Reply:
Actually I like the alternative below better.

Reply:
I literally had that before but thought fetch might be clear it's required without raising an argument error explicitly.  🤣  I'm fine with changing it.